We're not actually fetching in the port build; but this may be needed
during port updates to generate a tar of common mibs, so it's useful to
fix so it works if needed.

Index: generator/Makefile
--- generator/Makefile.orig
+++ generator/Makefile
@@ -129,7 +129,7 @@ mibs: mib-dir \
   $(MIBDIR)/EATON-OIDS.txt
 
 mib-dir:
-	@mkdir -p -v $(MIBDIR)
+	@mkdir -p $(MIBDIR)
 
 $(MIBDIR)/apc-powernet-mib:
 	@echo ">> Downloading apc-powernet-mib"
